onmt/translate/translator.py

```diff
@@ -58,14 +58,12 @@ class Translator(object):
        model (:obj:`onmt.modules.NMTModel`):
           NMT model to use for translation
        fields (dict of Fields): data fields
-       beam_size (int): size of beam to use
-       n_best (int): number of translations produced
-       max_length (int): maximum length output to produce
+       opt (obj): command line options
+       model_opt (obj): model options
        global_scores (:obj:`GlobalScorer`):
          object to rescore final translations
-       copy_attn (bool): use copy attention during translation
-       cuda (bool): use cuda
-       beam_trace (bool): trace beam search for debugging
```

Comment:
so we're removing the `beam_trace` option for now?

Reply:
no, but it's not an argument, it's an attribute passed through opt.